sql-cli/sql_cli/__main__.py

```diff
@@ -38,5 +41,53 @@ def generate(
     rprint("The DAG file", dag_file.resolve(), "has been successfully generated. 🎉")
 
 
+@app.command()
+def init(project_dir: Optional[str] = typer.Argument(None)) -> None:
```

Comment:
Can we use future annotations? And do we need the `typer.Argument` if you don't set it?
```suggestion
def init(project_dir: str | None = None) -> None:
```

Reply:
Also, you can use `Path` as type. Typer even has [additional features](https://typer.tiangolo.com/tutorial/parameter-types/path/) for it integrated. See [Path Validations](https://typer.tiangolo.com/tutorial/parameter-types/path/#path-validations).

Reply:
That's awesome, I love it, I'll use this!